tests/foreman/ui/test_host.py

```diff
@@ -1811,9 +1811,14 @@ def test_positive_delete_libvirt(
 
 
 @pytest.fixture
-def gce_client(download_cert):
+def gce_cert():
+    return json.loads(settings.gce.cert)
```

Comment:
Definitely would like to reduce the duplication going on here, `json.loads()` is called on the cert 5 times in different test modules. Single fixture should be able to download/load the cert, if its defined outside of the individual test modules.  See other comment about a shared session scoped fixture.